backend/model/ladybird_export.rb

```diff
@@ -0,0 +1,693 @@
+          :display_string => row[:display_string],
+          :first_term_type => row[:term_type]
+        }
+
+        @subjects[row[:archival_object_id]] << current
+      else
+        # we only care about the first term type
+      end
+    end
+  end
+
+  def prepare_digital_object_instance_flags
+    @has_digital_object_instances = []
+
+    Instance
+      .left_outer_join(:instance_do_link_rlshp, :instance_do_link_rlshp__instance_id => :instance__id)
+      .filter(:instance__archival_object_id => @ids)
+      .filter(Sequel.~(:instance_do_link_rlshp__digital_object_id => nil))
+      .select(Sequel.as(:instance__archival_object_id, :archival_object_id))
+      .distinct
+      .each do |row|
+      @has_digital_object_instances << row[:archival_object_id]
+    end
+  end
+
+  def local_record_id(row)
+    "/repositories/#{row[:repo_id]}/archival_objects/#{row[:archival_object_id]}"
+  end
+
+  def call_number(row)
+    JSON.parse(row[:resource_identifier]).compact.join(' ')
+  end
+
+  def box(row)
+    row[:top_container_indicator]
+  end
+
+  def barcode(row)
+    row[:top_container_barcode]
+  end
+
+  def folder(row)
+    row[:sub_container_folder]
+  end
+
+  def host_creator(row)
+    creators_for_resource(row[:archival_object_id])
+      .map{|row| (row[:person] || row[:corporate_entity] || row[:family] || row[:software])}
+      .join(NEW_LINE_SEPARATOR)
+  end
+
+  def host_title(row)
+    row[:resource_title]
+  end
+
+  def host_note(row)
+    breadcrumb_for_archival_object(row[:archival_object_id])
+  end
+
+  def note(row)
+    notes_for_archival_object(row[:archival_object_id])
+      .map{|type, content|
+        next if type == 'scopecontent'
+        next if type == 'accessrestrict'
+        next if type == 'prefercite'
+  
+        content.flatten
+      }
+      .flatten
+      .compact
+      .join(NEW_LINE_SEPARATOR)
+  end
+
+  def abstract(row)
+    notes_for_archival_object(row[:archival_object_id])
+      .map{|type, content|
+        next unless type == 'scopecontent'
+        content.flatten
+      }
+      .compact
+      .flatten
+      .join(NEW_LINE_SEPARATOR)
+  end
+
+  def title(row)
+    row[:archival_object_title]
+  end
+
+  def creator(row)
+    creators_for_archival_object(row[:archival_object_id])
+      .map{|row| (row[:person] || row[:corporate_entity] || row[:family] || row[:software])}
+      .join(NEW_LINE_SEPARATOR)
+  end
+
+  def language(row)
+    row[:archival_object_language]
+  end
+
+  def creation_date(row)
+    creation_dates_for_archival_object(row[:archival_object_id])
+      .map{|row| row[:expression] || [row[:begin], row[:end]].compact.join(' - ')}
+      .join(NEW_LINE_SEPARATOR)
+  end
+
+  def physical_description(row)
+    extents_for_archival_object(row[:archival_object_id])
+      .map{|row| "#{row[:number]} #{row[:extent_type]} (#{row[:portion]})" }
+      .join(NEW_LINE_SEPARATOR)
+  end
+
+  def name_subjects(row)
+    return if row[:archival_object_level] != 'item'
```

Comment:
remove all `return if row[:archival_object_level] != 'item'` and show for all nodes